source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -400,11 +396,8 @@ def _get_language(self, language):
             return Languages.english
 
     def get_language_parameter(self, request):
-        _language = request.get('_here_language', None)
-        if _language == None:
-            return self.language
-        else:
-            return self._get_language(_language.lower())
+        language = request.get('language', None)
```

Comment:
Should this default to `english`?
```suggestion
        language = request.get('language', "english")
```

Reply:
:+1: